src/main/java/org/opensearch/security/securityconf/impl/v6/ConfigV6.java

```diff
@@ -248,6 +248,15 @@ public String toString() {
                 + "]";
         }
 
+        @JsonAnySetter
+        public void unknownPropertiesHandler(String name, Object value) {
+            if (name.equals("transport_enabled")) {
+                LOGGER.warn(
+                    "Detected transport_enabled setting in config.yml file under an AuthcDomain. Since the transport client has been removed, this setting is now unnecessary/unsupported and therefore can be safely removed."
```

Comment:
Cleaned up this text, it would also be an improvement if the message could state which authenticator is associated with the value.
```suggestion
                    "In OpenSearch v2.0.0+ the setting 'transport_enabled' is deprecated, it should be removed from the authenticator named '{}'."
```